config/routes.rb

```diff
@@ -7,4 +7,5 @@
   resources :videos, only: %i[index show destroy create]
   resources :access_tokens
   resources :signature_keys
+  get 'total_watch_time' => 'stats#total_watch_time'
```

Comment:
The URL must be `GET /stats`

Ou can define it as a singular resource to keep it RESTful:

```suggestion
  resource :stats, only: :show
```

Documentation: https://guides.rubyonrails.org/routing.html#singular-resources